rllib/examples/rl_modules/classes/vpg_torch_rlm.py

```diff
@@ -11,11 +11,20 @@ class VPGTorchRLModule(TorchRLModule):
     """
 
     def setup(self):
-        # You have access here to the following already set attributes:
-        # self.observation_space
-        # self.action_space
-        # self.inference_only
-        # self.model_config  # <- a dict with custom settings
+        """Use this method to create all the model components that you require.
+
+        Feel free to access the following useful properties in this class:
+        - `self.model_config`: The config dict for this RLModule class,
+        which should contain flexible settings, for example: {"hiddens": [256, 256]}.
+        - `self.observation|action_space`: The observation and action space that
+        this RLModule is subject to. Note that the observation space might not be the
+        exact space from your env, but that it might have already gone through
+        preprocessing through a connector pipeline (for example, flattening,
+        frame-stacking, mean/std-filtering, etc..).
```

Comment:
Note: I think we should, at some point, disambiguate the word `observation_space` be changing it to `input_space` or something similar.